src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -1270,12 +1244,18 @@ private Task<INetworkInterface> AssociateNicWithNetworkSecurityGroupAsync(INetwo
                 () => networkInterface.Update().WithExistingNetworkSecurityGroup(networkSecurityGroup).ApplyAsync()
             );
 
-        private Task<FirewallRule> AddMySQLFirewallRuleForNicAsync(IVirtualMachine vm, IMySQLManagementClient sqlManagementClient)
+        private Task ExecuteQueriesOnAzureMySqlDb(ConnectionInfo sshConnectionInfo, IMySQLManagementClient mySQLServer)
         => Execute(
-            $"Adding MySQL Firewall Rule for VM NIC. IP: {vm.GetPrimaryPublicIPAddress().IPAddress}",
-            () => sqlManagementClient.FirewallRules.CreateOrUpdateAsync(configuration.ResourceGroupName, MySqlServerName,
-                    "AllowVMNic", new FirewallRule(vm.GetPrimaryPublicIPAddress().IPAddress, vm.GetPrimaryPublicIPAddress().IPAddress))                
-            );
+            $"Executing scripts on cromwell_db.",
+            async () => {
+                await ExecuteCommandOnVirtualMachineAsync(sshConnectionInfo, $"sudo apt install -y mysql-client");
+                var initScript = Utility.PersonalizeContent(new[]
+                {
+                    new Utility.ConfigReplaceRegExItemText("^CREATE USER 'cromwell'@'%'.*$", String.Empty, RegexOptions.Multiline),
```

Comment:
Rather than replacing the text in init script, how about just executing here:
`GRANT ALL PRIVILEGES ON cromwell_db.* TO 'cromwell' WITH GRANT OPTION; FLUSH PRIVILEGES;`

Reply:
I'm confused, the line we're replacing is `CREATE USER 'cromwell'@'%' IDENTIFIED BY 'cromwell';` and we need to create the localhost user. so what we end up executing w/ azure is 
```
CREATE USER 'cromwell'@'localhost' IDENTIFIED BY 'cromwell';
GRANT ALL PRIVILEGES ON cromwell_db.* TO 'cromwell'@'localhost' WITH GRANT OPTION;
GRANT ALL PRIVILEGES ON cromwell_db.* TO 'cromwell'@'%' WITH GRANT OPTION;
FLUSH PRIVILEGES;
```
The RegEx solution was Blair's suggestion so we could keep the Create User line in for the docker deployment.

Reply:
Ideally Cromwell should not use server admin user. I suggest creating a separate user just for Cromwell with access to cromwell_db only.

Reply:
Created issue for further follow up work from this PR: https://github.com/microsoft/CromwellOnAzure/issues/374